Modern_Tides.json

```
@@ -0,0 +1,5 @@
+{
+  "title": "Modern Tides",
+  "author": "Ryan Rutishauser",
+  "stylesheet": "https://github.com/RVCC-IDMX/style-mobile-ryan-rutishauser/blob/main/public/style.css"
```

Update to "raw" stylesheet link :)
```suggestion
  "stylesheet": "https://raw.githubusercontent.com/RVCC-IDMX/style-mobile-ryan-rutishauser/main/public/style.css"
```